platine-management-service/src/main/java/fr/insee/survey/datacollectionmanagement/metadata/service/impl/OwnerServiceImpl.java

```diff
@@ -32,35 +31,14 @@ public Owner insertOrUpdateOwner(Owner owner) {
 
         try {
             Owner ownerBase = findById(owner.getId());
```

Comment:
findById could return an Optional, you won't need the try/catch in this case. The save method could be after the try/catch